src/main/kotlin/com/msd/robot/application/RobotApplicationService.kt

```diff
@@ -69,18 +69,7 @@ class RobotApplicationService(
                 eventSender.handleException(fe, command)
             }
         }
-        robotPlanetPairs.forEach { (command, pair) ->
-            val moveEventId = successEventSender.sendMovementEvent(pair.first, pair.second.movementDifficulty, command.transactionUUID)
-            eventSender.sendEvent(
-                ItemMovementEventDTO(
-                    true,
-                    "Item usage successful",
-                    moveEventId
-                ),
-                EventType.ITEM_MOVEMENT,
-                command.transactionUUID
-            )
-        }
+        successEventSender.sendAllMovementEvents(robotPlanetPairs)
```

Comment:
You moved this event to another method but there are still other events that are being thrown in directly in the methods rather than from the `SuccessEventSender`. What about them? Is it intended that they're not moved to another class?